routes/schemas.js

```diff
@@ -114,12 +114,23 @@ module.exports = {
 				.keys()
 				.pattern(
 					/^(IMPRESSION|CLICK)$/,
-					Joi.object({ min: numericString.default('0'), max: numericString.default('0') })
+					Joi.object({ min: numericString.required(), max: numericString.required() })
```

Comment:
why does this no longer have a default?

Reply:
This because if it has default '0', then the behavior becomes unreliable if an advertiser specifies one property and omits the next. e.g. specifies `min` omits `max`